### DISCUSSION TITLE
Use AWS Portfolio to cover computation costs

### DISCUSSION THREAD

**Discussion by charanjit-singh** [reworkd/AgentGPT#392] [Ideas]
You can join nvidia inception program and avail for AWS portfolio to cover infrastructure costs.

Around ~250,000 USD worth of credits.

**Comment by awtkns** [comment]
> You can join nvidia inception program and avail for AWS portfolio to cover infrastructure costs.
> 
> Around ~250,000 USD worth of credits.
>

That's a great idea! For now though the majority of our bill is from OpenAi though 😢
